group_name: manipulator
queries_number: 10   # number of tuple start-goal
query_prefix: "DOF3_cubes"
repetitions: 30  #number of repetitions for each plan
pipeline_ids:
- dirrt
- ompl

planning_times: [0.1,0.2,0.3,0.4,0.5,0.6,0.7,0.8,0.9,1.0,1.5,2.0,3.0,4.0,5.0]
planner_ids:
  dirrt:
  - mixed_strategy
  - mixed_strategy_extend
  - informed_rrt

  ompl:
  - BiTRRT
  - RRTstar
  - BFMT
  - LazyPRMstar
